mvn-defaults/pom.xml

```diff
@@ -547,7 +548,7 @@
         <plugin>
           <groupId>org.antlr</groupId>
           <artifactId>antlr4-maven-plugin</artifactId>
-          <version>4.9.2</version>
```

Comment:
I have a change in my post-merge fixups for these locally. I'll merge or close this PR soon.